PlasmaResearch/plasma_batch_auction.tex

```diff
@@ -607,7 +609,7 @@ \section{An elaboration on the snark mechanism}
 The public key used to encrypt all orders will be denoted by $\text{key}_p$.
 In order to run the proofs the most efficient, we need to calculate a special hash:
 \begin{equation}
-h = \text{sha256}(M_R(\mathcal{B}_O),\mathcal{H}_V, \text{sha256}(\beta), M_R(\vec{p}), \text{key}_p) 
+h = \text{sha256}(sha256(\mathcal{B}_O),\mathcal{H}_V, \text{sha256}(\beta), sha256(\vec{p}), \text{key}_p) 
```

Comment:
Did you also want to put a sha256 onto \mathcal{H}_V? I think we should also declare a math operator for sha256. I think that I will make a bunch of these changes myself and you can look at them after...